js/graviex.js

```diff
@@ -0,0 +1,982 @@
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'network': undefined,
+            'address': undefined,
+            'addressTo': undefined,
+            'addressFrom': undefined,
+            'tag': undefined,
+            'tagTo': undefined,
+            'tagFrom': undefined,
+            'type': type,
+            'amount': amount,
+            'currency': code,
+            'status': status,
+            'updated': undefined,
+            'internal': undefined,
+            'fee': fee,
+        };
+    }
+
+    parseTransactionStatus (status) {
+        const statuses = {
+            'initiated': 'pending',
+            'needs_create': 'pending',
+            'credited': 'ok',
+            'confirmed': 'ok',
+        };
+        return this.safeString (statuses, status, status);
+    }
+
+    async fetchDeposit (id, code = undefined, params = {}) {
+        if (id === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchDeposit() requires a txid argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+            'txid': id,
+        };
+        const response = await this.privateGetDeposit (this.extend (request, params));
+        // {
+        //   "id": 4831985,
+        //   "currency": "ltc",
+        //   "amount": "0.14108256",
+        //   "fee": "0.0",
+        //   "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //   "uid": null,
+        //   "created_at": "2022-02-01T18:10:32+03:00",
+        //   "confirmations": "6",
+        //   "done_at": null,
+        //   "state": "accepted"
+        // }
+        // const data = this.safeValue (response, 'data', {});
+        // const deposit = this.safeValue (data, 'deposit', {});
+        return this.parseTransaction (response);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+        }
+        const response = await this.privateGetDeposits (params);
+        // const data = this.safeValue (response, 'data', {});
+        // const deposits = this.safeValue (response);
+        // [
+        //     {
+        //         "id": 4831985,
+        //         "currency": "ltc",
+        //         "amount": "0.14108256",
+        //         "fee": "0.0",
+        //         "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //         "uid": null,
+        //         "created_at": "2022-02-01T18:10:32+03:00",
+        //         "confirmations": "6",
+        //         "done_at": null,
+        //         "state": "accepted"
+        //     }
+        // ]
+        return this.parseTransactions (response, currency, since, limit);
+    }
+
+    parseOrderStatus (status) {
+        const statuses = {
```

Comment:
i'm not really sure what this mapping is, firstly the order responses are on the right instead of the left and secondly the stuff on the left is not even the unified ccxt statuses